agent/otel/bridgeservice.go

```diff
@@ -0,0 +1,46 @@
+package otel
+
+import (
+	"github.com/ns1labs/orb/agent/policies"
+	"strings"
+)
+
+type AgentBridgeService interface {
+	RetrieveAgentInfoByPolicyName(policyName string) (*AgentDataPerPolicy, error)
+}
+
+type AgentDataPerPolicy struct {
+	PolicyID  string
+	Datasets  string
+	OrbTags   string
+	AgentTags string
+}
+
+var _ AgentBridgeService = (*bridgeService)(nil)
+
+type bridgeService struct {
+	policyRepo policies.PolicyRepo
+	OrbTags    string
+	AgentTags  string
+}
+
+func NewBridgeService(policyRepo *policies.PolicyRepo) *bridgeService {
+	return &bridgeService{
+		policyRepo: *policyRepo,
+		OrbTags:    "testing-orb",
```

Comment:
should be removed?

Reply:
This was a preparation test to inject orb tags into requestMetrics otlp object. I belive that the intention is to conclude it tomorrow.

Reply:
I am leaving this for now, once my addition to pass that information to the agent, is complete, we can switch, better than nothing, at least we can start testing, expecting values there